pkg/deploy/api/test/ok.go

```diff
@@ -22,7 +22,9 @@ const (
 func OkDeploymentConfig(version int64) *deployapi.DeploymentConfig {
 	return &deployapi.DeploymentConfig{
 		ObjectMeta: kapi.ObjectMeta{
-			Name: "config",
+			Name:      "config",
+			Namespace: kapi.NamespaceDefault,
```

Comment:
`fake.NewSimpleClientset` requires to have namespace set, it panicked without it. The unit tests are green already ;)